bedrock/releasenotes/models.py

```diff
@@ -124,7 +124,7 @@ def refresh(self):
         release_objs = []
         rn_path = os.path.join(settings.RELEASE_NOTES_PATH, 'releases')
         with transaction.atomic(using=self.db):
-            self.all().delete()
+            self.get_queryset(True).delete()
```

Comment:
would you mind changing the param to `include_drafts=True` so that it's more obvious why this is necessary?